hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/query/QueryWorkflowImplTest.java

```diff
@@ -464,526 +470,140 @@ void testInactivePlatformFails(@Mock CurrentPlatformStatus localCurrentPlatformS
-    //        assertThat(response.fileGetInfo()).isNotNull();
-    //        final var header = response.fileGetInfoOrThrow().header();
-    //        assertThat(header.nodeTransactionPrecheckCode()).isEqualTo(NOT_SUPPORTED);
-    //        assertThat(header.responseType()).isEqualTo(COST_ANSWER);
-    //        assertThat(header.cost()).isZero();
-    //        verify(opCounters).countReceived(NetworkGetExecutionTime);
-    //        verify(opCounters, never()).countAnswered(NetworkGetExecutionTime);
-    //    }
-    //
-    //    @Test
-    //    void testQuerySpecificValidationFails()
-    //            throws PreCheckException, InvalidProtocolBufferException {
-    //        // given
-    //        doThrow(new PreCheckException(ACCOUNT_FROZEN_FOR_TOKEN))
-    //                .when(dispatcher)
-    //                .validate(state, query);
-    //        final var responseBuffer = ByteBuffer.allocate(BUFFER_SIZE);
-    //
-    //        // when
-    //        workflow.handleQuery(ctx, requestBuffer, responseBuffer);
-    //
-    //        // then
-    //        final var response = parseResponse(responseBuffer);
-    //        assertThat(response.fileGetInfo()).isNotNull();
-    //        final var header = response.fileGetInfoOrThrow().header();
-    //        assertThat(header.nodeTransactionPrecheckCode()).isEqualTo(ACCOUNT_FROZEN_FOR_TOKEN);
-    //        assertThat(header.responseType()).isEqualTo(ANSWER_ONLY);
-    //        assertThat(header.cost()).isZero();
-    //        verify(opCounters).countReceived(FileGetInfo);
-    //        verify(opCounters, never()).countAnswered(FileGetInfo);
-    //    }
-    //
-    //    @Test
-    //    void testPaidQueryWithFailingSubmissionFails()
-    //            throws PreCheckException, InvalidProtocolBufferException {
-    //        // given
-    //        when(handler.requiresNodePayment(ANSWER_ONLY)).thenReturn(true);
-    //        doThrow(new PreCheckException(PLATFORM_TRANSACTION_NOT_CREATED))
-    //                .when(submissionManager)
-    //                .submit(txBody, payment.toByteArray(), ctx.txBodyParser());
-    //        final var responseBuffer = ByteBuffer.allocate(BUFFER_SIZE);
-    //
-    //        // when
-    //        workflow.handleQuery(ctx, requestBuffer, responseBuffer);
-    //
-    //        // then
-    //        final var response = parseResponse(responseBuffer);
-    //        assertThat(response.fileGetInfo()).isNotNull();
-    //        final var header = response.fileGetInfoOrThrow().header();
-    //        assertThat(header.nodeTransactionPrecheckCode())
-    //                .isEqualTo(PLATFORM_TRANSACTION_NOT_CREATED);
-    //        assertThat(header.responseType()).isEqualTo(ANSWER_ONLY);
-    //        assertThat(header.cost()).isZero();
-    //        verify(opCounters).countReceived(FileGetInfo);
-    //        verify(opCounters, never()).countAnswered(FileGetInfo);
-    //    }
```

Comment:
These tests were duplicates. There's another test for each one of these with the same name